packages/hint-minified-js/tests/tests.ts

```diff
@@ -1,5 +1,6 @@
 import { HintTest, testHint } from '@hint/utils-tests-helpers';
 import { test } from '@hint/utils';
+import { Severity } from 'hint';
```

Comment:
Can you double-check this doesn't increase the size of the browser extension? I think we need a deep import here. That said, this probably becomes a non-issue once we rebase on top of @sarvaje's flatten PR so long as we update this reference to be pulled from the new `@hint/utils-types` package.

Reply:
Nevermind - just realized this is only in test code so it won't be part of the bundle. Your import in the implementation already does a deep import.